### DISCUSSION TITLE
How do I set up custom SMTP for auth? I am using Hostinger as my mail provider.

### DISCUSSION THREAD

**Discussion by LublubXT** [supabase/supabase#29041] [Questions]
I am trying to set up the custom SMTP for my SaaS app. I am using Hostinger as my email provider, and the SMTP server is “smtp.hostinger.com” When I try to sign up with the SaaS app I am making I get the error, “Error: Error sending confirmation mail” I sign up with Supabase using my GitHub account, and I don't know what to put in the username and password section under the “Custom SMTP” section. I looked at Supabase's documentation, but I couldn't find the info I needed.
![Screenshot 2024-09-02 182057](https://github.com/user-attachments/assets/177a366a-2625-42fd-875c-e999869f45fa)
![Screenshot 2024-09-02 182005](https://github.com/user-attachments/assets/928d8b1e-2d76-41db-9f8f-8687e9e3c3c8)
Also, why do I get the warning like the following, when I already filled in all the fields?
![Screenshot 2024-09-02 181943](https://github.com/user-attachments/assets/9458548f-b5cc-4153-9615-09603abaaf78)

**Comment by GaryAustin1** [comment]
You'll have to check with Hostinger on the setup requirements assuming it does support being an SMTP server for sending out emails.

You can look in the dashboard auth log for maybe more info on the error back from your provider.

The all fields would include the first two you don't show on your screen for sender email and name.

**Comment by n3twrks** [comment]
Any solution found ?
